code/controllers/subsystem/mapping.dm

```diff
@@ -590,3 +591,10 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 		isolated_ruins_z = add_new_zlevel("Isolated Ruins/Reserved", list(ZTRAIT_RESERVED = TRUE, ZTRAIT_ISOLATED_RUINS = TRUE))
 		initialize_reserved_level(isolated_ruins_z.z_value)
 	return isolated_ruins_z.z_value
+
+/datum/controller/subsystem/mapping/proc/spawn_maintenance_loot()
+	for(var/obj/effect/spawner/lootdrop/maintenance/spawner as anything in GLOB.maintenance_loot_spawners)
+		CHECK_TICK
+
+		spawner.spawn_loot()
+		spawner.invisibility = INVISIBILITY_ABSTRACT
```

Comment:
I feel like this should be a property of `/obj/effect/spawner/lootdrop/maintenance`

Reply:
I wanted them to be visible in pre-round, as a sort of "hey look guys at the map you live in" for the observers, and being helpful for debugging when I was working with them.

If you feel strongly I can change it, but I thought it was cute. 🤷

Reply:
I have added a `hide()` proc to the maintenance loot spawners, which hides from non-observers and set the transparency to low.